### DISCUSSION TITLE
MS-Office>2007 and Libreoffice protected files

### DISCUSSION THREAD

**Discussion by carliedu** [openssl/openssl#21672] [Q&A]
I'm curious to know, why openssl still cannot decrypt CDFV2 Encrypted docx, xlsx, etc. files, created by MS-Office>2007 and Libreoffice 7.

**Comment by paulidale** [comment]
I can think of two reasons:
1. nobody was written the code for the project
2. there doesn't seem a lot of utility in having it.  I.e. it doesn't prevent one from needing MS-Office or Libreoffice to use the files (since the are "their" format) and if you've got them, they are perfectly able to decrypt the files.
